app_flutter/lib/logic/links.dart

```diff
@@ -25,11 +25,6 @@ List<CocoonLink> createCocoonLinks(BuildContext context) {
       icon: const Icon(Icons.build),
       action: () => Navigator.pushReplacementNamed(context, BuildDashboardPage.routeName),
     ),
-    CocoonLink(
-      name: 'Benchmarks',
-      icon: const Icon(Icons.show_chart),
-      action: () => launch('/benchmarks.html'),
-    ),
     CocoonLink(
       name: 'Framework Benchmarks on Skia Perf',
```

Comment:
You can drop the skia perf wording now (eg "Framework Benchmarks")

Reply:
Done.

> LGTM when comments are addressed.
> 
> You should schedule an announcement to the team so there are no surprises.

A notice email will be sent out by today to schedule the deprecation by end of tomorrow.
This PR will not be submitted today.